packages/flutter_tools/lib/src/project.dart

```diff
@@ -552,10 +552,28 @@ class AndroidProject extends FlutterProjectPlatform {
     return imperativeMatch || declarativeMatch;
   }
 
+  /// Gets top-level Gradle build file.
+  /// See https://developer.android.com/build#top-level.
+  ///
+  /// It can be written in Groovy (build.gradle) or Kotlin (build.gradle.kts).
```

Comment:
Please include in dart doc the behavior when both files are present.

Reply:
done in d30b4da17b0ceb0c50670f5b2907328ceee91771